docs/probe.md

```diff
@@ -0,0 +1,177 @@
+# Probes and Meters: Examples for Adding to Code
+For an introduction to `Probe`s and `Meter`s, please refer to [Measurement Overview](./metrics.md).
+
+## Context
+To monitor particular aspects of an `armory run` session, the user needs to know the following factors:
+- What am I measuring?
+- When should I measure it?
+- Where should my custom monitoring script go?
+
+We assume the user is modifying code for models or attacks and wishes to use `Probe`s and `Meter`s to monitor certain variables within the code.
+
+Recall the steps for a minimal working example (in [Measurement Overview](./metrics.md)):
```

Comment:
in `metrics.md`, where we say `Meters and writers can be instantiated in your initial setup`, can we link to the user_init docs here?

Reply:
Resolved with dd462e2b